nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -220,6 +221,13 @@
             .allowableValues(CLIENT_NONE, CLIENT_WANT, CLIENT_NEED)
             .defaultValue(CLIENT_NONE.getValue())
             .build();
+    public static final PropertyDescriptor TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Timeout")
+            .description("The timeout to set when handling requests")
```

Comment:
@markap14 I believe this is something specific to jetty. As reported on the mailing list/JIRA, when the timeout expires, there is no response sent to the client, instead there is a new request received in the processor with some differences:

> the http.context.identifier and the FlowFile UUID has changed, and the http.dispatcher.type has changed from REQUEST to ERROR

If the timeout is not set properly (or if there is a problem in the flow and response is never sent), it could generate an infinite loop leading to the problem reported

>  If I then leave the system alone for much longer than the Request Timeout parameter in the StandardHttpContextMap and then attempt a request, I get a 503 response saying that the queue is full and no requests are allowed. No requests are allowed at all until I delete and recreate the Map.